frontend/src/v5/ui/routes/viewer/tickets/ticketsList/ticketItem/ticketItem.component.tsx

```diff
@@ -46,6 +50,14 @@ export const TicketItem = ({ ticket, onClick, selected }: TicketItemProps) => {
 	} = ticket;
 	const riskLevel = ticket.modules?.safetibase?.['Level of Risk'];
```

Comment:
**Not a change request**
When you have to access deep properties and need using several optional chaining operators, you may use the `get` method from lodash which does the trick for you. So the above could be written as `_.get(ticket, 'modules.safetibase.Level of Risk');`
Just so you know, but again, this is not a change request

Reply:
Yeah, I think it depends on the context it's being used. In this case the line lengths are about the same as you have to specify the full path so that custom properties with the same name aren't picked. I do tend to forget about `get` though, so it's good for me to remember